IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md

```diff
@@ -0,0 +1 @@
+Existence and type or properties of filter segment installed in the casing of a borehole.
```

Comment:
Is it just me having understanding issues with this description?

Reply:
No. Pinging @j-weil

Reply:
I changed to "Identification of filter..."